libs/order/src/models/order-payment.ts

```diff
@@ -0,0 +1,19 @@
+export interface DaffOrderPayment {
```

Comment:
This should be a union type of the various possible payment methods., not all payment methods should be forced to a single type.

Reply:
So we need a type for every payment method? CC, paypal, etc.?